### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Prachi-Chaturvedi** [TIL-Vadodara/til.camp#32]
The very first issue with Accessibility is usage of JavaScript. The usage of script is allowed but it should be limited to certain elements or features. Entire site on JavaScript is a very bad practice as it makes the websites completely inaccessible once the scripting is disabled. Also, main issue with usage of scripting is inability or difficulty navigating using a keyboard or assistive technology.

**Comment by nayaabkhan** [comment]
Thanks @Prachi-Chaturvedi for the review. I'll see how we can have site working even with JS disabled.

@lakhansamani I might migrate the site from CRA to Gatsby for this. Just a heads-up. 😉

**Comment by lakhansamani** [comment]
Thanks @Prachi-Chaturvedi 

@nayaabkhan lets do it, also wanted to confirm from @Prachi-Chaturvedi if we use Server Side Rendering with Javascript technologies, is it more accessible?

In that case we can consider NextJS

**Comment by nayaabkhan** [comment]
Next.js brings in more infrastructure needs with itself. As we will require to run it somewhere.

Gatsby can generate static site from React code itself so the site would still work without JS. Gatsby is like Jekyll for React.

**Comment by nayaabkhan** [comment]
Reference: https://youtu.be/l95VFLj3e2w
